i18n/en.json

```diff
@@ -1815,6 +1815,53 @@
   "admin.reload.reloadDescription.featureName": "Reload Configuration from Disk",
   "admin.reload.reloadDescription.recycleDatabaseConnections": "Environment > Database > Recycle Database Connections",
   "admin.reload.reloadFail": "Reloading unsuccessful: {error}",
+  "admin.reporting.workspace_optimization.access.description": "Web server settings could be affecting access.",
+  "admin.reporting.workspace_optimization.access.site_url.description": "Your webserver settings are not passing a live URL test, this would prevent users from accessing this workspace, we recommend updating your settings.",
```

Comment:
```suggestion
  "admin.reporting.workspace_optimization.access.site_url.description": "Your web server configuration isn't passing a live URL test which means your workspace may not be accessible to users. [Update web server settings].",
```

Reply:
[Update web server settings] link would take the user to https://docs.mattermost.com/configure/configuration-settings.html#environment

Reply:
@cwarnermm we do have a CTA button living in the accordions, so I would rather add the link to that button, ok?